project/AppDependencies.scala

```diff
@@ -4,18 +4,23 @@ object AppDependencies {
 
   private val bootstrapVersion = "8.3.0"
   private val hmrcMongoVersion = "1.6.0"
+  val scalamockVersion         = "5.2.0"
 
   val compile = Seq(
     play.sbt.PlayImport.ws,
     "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "8.5.0",
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapVersion,
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % hmrcMongoVersion
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % hmrcMongoVersion,
+    "org.mockito"       %% "mockito-scala"              % "1.17.12",
```

Comment:
mockito and scalamock are only needed for tests, please remove these from the src/compile list

Reply:
This is fixed now